speechbrain/augment/preparation.py

```diff
@@ -51,6 +53,7 @@ def prepare_dataset_from_URL(URL, dest_folder, ext, csv_file, max_length=None):
         prepare_csv(filelist, csv_file, max_length)
 
 
+@main_process_only
 def prepare_csv(filelist, csv_file, max_length=None):
```

Comment:
is this function often called outside of `prepare_dataset_from_URL` ?

Reply:
I doubt it, but it shouldn't hurt anything to add the decorator here just in case anyone does.